rust/oak/src/lib.rs

```diff
@@ -64,9 +67,14 @@ where
     }
 }
 
+/// Handle used to identify read or write channel halves.
+///
+/// These handles are used for all host function calls.
 pub type Handle = u64;
 
-/// Map an OakStatus to the nearest available std::io::Result.
+/// Map an [`OakStatus`] value to the nearest available std::io::Result.
+///
+/// [`OakStatus`]: proto/oak_api/enum.OakStatus.html
```

Comment:
Can this be

```
[`OakStatus`]: proto::oak_api::OakStatus
```

(and below)?

see https://github.com/rust-random/rand/blob/master/src/lib.rs#L210

Reply:
Oh, neat, I didn't know you could do that.  Fixed.